agent-kit/skills/claude-hook-authoring/scripts/scaffold-hook.sh

```
@@ -0,0 +1,481 @@
+  exit 1
+fi
+
+if [[ ! " ${VALID_EVENTS[*]} " =~ " ${EVENT_TYPE} " ]]; then
+  echo -e "${RED}Error: Invalid event type: $EVENT_TYPE${NC}"
+  echo "Valid events: ${VALID_EVENTS[*]}"
+  exit 1
+fi
+
+# Validate language
+VALID_LANGUAGES=(bash typescript python)
+if [[ ! " ${VALID_LANGUAGES[*]} " =~ " ${LANGUAGE} " ]]; then
```

Shellcheck (0.11.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)